src/types.ts

```diff
@@ -67,6 +67,16 @@ export interface PaymentNotificationData {
   };
 }
 
+export interface AuthResponseObject {
+  reason?: string;
+  status: string;
```

Comment:
btw. some people were saying we should simply return what the service returns. 
Now they miss some data in the webln.lnurl(auth-str) response.

Reply:
Extended it. Have a look. Not sure if that's what you meant.